CMakeLists.txt

```diff
@@ -18,6 +18,15 @@
 #   02110-1301, USA.
 message("Entering ${CMAKE_CURRENT_SOURCE_DIR}")
 
+if(${CMAKE_VERSION} VERSION_GREATER_EQUAL "3.31")
```

Comment:
Do we still need this one?
We normally have already done the upgrade to 3.31 in #2169

Reply:
Tested and work well here. no warnings or whatever. so closing